Pokedex/Detail/DetailViewController.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  DetailViewController.swift
+//  Pokedex
+//
+//  Created by krikaz on 6/9/20.
+//  Copyright © 2020 thewire. All rights reserved.
+//
+
+import UIKit
+
+final class DetailViewController: UIViewController {
+    
+    var pokemon: String?
+    private let viewModel = DetailViewModel()
```

Comment:
Rather than having your VC holding on `var pokemon: String?` (which should be named `pokemonName`), put it in your VM because that's where it will be used. Even better, you can use dependency injection (I'll let you search for the definition) and pass it in the initializer of your DetailViewModel.